container-disc/src/main/java/com/yahoo/container/jdisc/metric/MetricUpdater.java

```diff
@@ -121,8 +119,8 @@ public void run() {
         private final Timer timer = new Timer();
 
         @Override
-        public void schedule(Runnable runnable, TemporalAmount frequency) {
-            long frequencyMillis = frequency.get(ChronoUnit.MILLIS);
+        public void schedule(Runnable runnable, Duration frequency) {
```

Comment:
Since it's a duration of time, I think it should be 'period' rather than frequency. Frequency is the inverse.

Reply:
Agree, I will fix in upcoming PR.